explainer.md

```diff
@@ -605,6 +605,29 @@ xrSession.addEventListener('resetpose', xrSessionEvent => {
 });
 ```
 
+### Page navigation
+
```

Comment:
I see that headset activation was removed. While it doesn't have to be in this PR, I think we should know how we want to handle that (i.e., have a companion PR) before we effectively commit to an API shape for "auto presentation" scenarios.